examples/benchmarks/bubblebench/simple-tree/CHANGELOG.md

```diff
@@ -1,8 +1,4 @@
-# @fluid-example/bubblebench-common
-
-## 2.0.0-rc.2.0.0
-
-Dependency updates only.
+# @fluid-example/bubblebench-simple-tree
 
 ## 2.0.0-rc.1.0.0
```

Comment:
given that this wasn't included in rc.1, might as well remove this line too.

Reply:
I guess the whole CHANGELOG is not needed